news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "PrestaShop 1.7 minor release lifecycle"
+subtitle: "The journey of a minor release of PS 1.7, step by step"
+date:   2020-03-12 16:00:00
+authors: [ MathieuFerment ]
+icon: icon-code
+tags:
+ - releases
+ - 1.7
+ - beta
+---
+
+
+The journey of the release of PrestaShop 1.7.7.0 release has started. This moment is a nice opportunity to explain fully the PrestaShop minor releases lifecycle.
+
+# The journey begins: Kanban and scope
+
+Each minor version is defined by a feature scope, and this scope is defined as a number of github issues that we put into a [Kanban](https://help.github.com/en/github/managing-your-work-on-github/about-project-boards) (see [1.7.7 Kanban](https://github.com/PrestaShop/PrestaShop/projects/7)). How these issues are picked or sorted is the responsibility of the Product Team which spends a lot of time gathering feedback about all PrestaShop users to make sure the next minor version addresses the most important matters and needs.
+
+For example 1.7.7.0 scope contained, but not only:
+- Compatibility with php 7.3
+- Migration and rework of Back-office Order pages
+- Advanced currencies management
+
+Although it is very hard to estimate the size of this scope, we try to size it in order for the development phase to last 4 months[2].
```

Comment:
Shouldn't [1] come before [2]?

Reply:
Indeed, as I rewrote the article I inverted the references. I will fix that